content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -31,7 +31,7 @@ check the documentation for that version of Kubernetes.
 
 ## API
 
-The new `resource.k8s.io/v1alpha1` {{< glossary_tooltip text="API group"
+The new `resource.k8s.io/v1alpha2` {{< glossary_tooltip text="API group"
```

Comment:
```suggestion
The `resource.k8s.io/v1alpha2` {{< glossary_tooltip text="API group"
```
avoid “new” in docs (it's fine in changelogs). Docs should aim to be [timeless](https://developers.google.com/style/timeless-documentation)